logging/logging.go

```diff
@@ -47,7 +47,7 @@ func NewZapLoggerConfig() zap.Config {
 			MessageKey:     "msg",
 			StacktraceKey:  "stacktrace",
 			LineEnding:     zapcore.DefaultLineEnding,
-			EncodeLevel:    zapcore.CapitalColorLevelEncoder,
```

Comment:
I can follow up offline, but I'm curious which pieces logged with color and which did not such that it became preferable to have no color at all.

Reply:
Stuff in goutils, like the grpc server. You should pull down the PR and revert that line and run viam-server locally. I'm happy to make a compromise somewhere, but I'm not sure what suggestions there are from others that have seen the mix.

Reply:
Cool I'll take a look offline; thanks.

Reply:
Yeah I see it; I guess I just took the mix of color and no color as "normal", but I'm fine with no color at all for now.